templates/auth_error.html

```diff
@@ -2,6 +2,11 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+
+    {% if config.get("B2C_RESET_PASSWORD_AUTHORITY") and "AADB2C90118" in result.get("error_description") %}
```

Comment:
@jennyf19 To continue from [your earlier comment](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/4/files#r341422631), here in the latest commit we added the ResetPassword user flow behavior.

Reply:
cool. thanks

---
In reply to: [343859221](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/4#discussion_r343859221) [](ancestors = 343859221)